docs/source/plugin_dev/classes/ServerInterface.rst

```diff
@@ -49,8 +49,6 @@ If args or kwargs contains `RText <../api.html#rtext>`__ element, then the resul
 
 If the translation key is not recognized, the return value will be the translation key itself
 
-See `here <../basic.html#translation>`__ for the ways to register translations for your plugin
```

Comment:
What is this for?

Reply:
This doc translation was finished and regenerated comments by sphinx before #123 is checked. Actually I didn't edit this line manually

Reply:
idc, just fix this

Reply:
If this affects document format, i'll regenerate and edit this file from current `dev` branch